android/src/main/java/net/no_mad/tts/TextToSpeechModule.java

```diff
@@ -61,14 +61,15 @@
     private final Executor executor = Executors.newSingleThreadExecutor();
     private final ReentrantLock lock = new ReentrantLock();
     private final Condition condition = lock.newCondition();
-    private boolean isTtsCompleted = false;
+    private final Condition conditionStopped = lock.newCondition();
+    private boolean isTtsCompleted = true;
```

Comment:
Since this is the default state, would `isTtsIdle` better reflect this flag?

Reply:
changed to use isTtsIdle in commit [06e82b1](https://github.com/tomtom-international/react-native-tts/pull/2/commits/06e82b1674e521a45c3b0ab4e366a8313d46795d)